test/e2e/framework/pv/pv.go

```diff
@@ -865,3 +865,12 @@ func WaitForPVCFinalizer(ctx context.Context, cs clientset.Interface, name, name
 	}
 	return err
 }
+
+// GetDefaultFSType returns the default fsType, by default it's ext3 unless
+// it's windows, if it's windows the default is ntfs
+func GetDefaultFSType() string {
+	if framework.NodeOSDistroIs("windows") {
+		return "ntfs"
+	}
+	return "ext3"
```

Comment:
is default ext4 or ext3?

Reply:
In one of the test it was set to ext3 so I thought it was the default

https://github.com/mauriciopoppe/kubernetes/blob/14c25eed8d07acdfaf882674f58fd2aa4cc7afe6/test/e2e/storage/persistent_volumes-gce.go#L95

Reply:
I think default should be ext4

Reply:
I've updated the default to be ext4